bin/dump-bir

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This may be done better in Main.lean, but the way it's written it's not ideal. Maybe we need those functions to be split up a bit and return String so we can combine e.g. gather and compile more easily.

Reply:
Updated to move it to Lean rather than Python